api/src/gotEnzymes/search.js

```diff
@@ -3,7 +3,14 @@ import sql from 'gotEnzymes/db';
 const search = async searchTerm => {
   const term = searchTerm.toLowerCase();
   const fuzzyQuery = sql`
-    select type, id, match, ${sql`similarity(match, ${term})`} as score
+    select
+      type,
+      id,
+      match,
+      ${sql`similarity(
+        ${sql`replace(match, '.', 'a')`},
+        ${sql`replace(${term}, '.', 'a')`}
```

Comment:
I'm wondering if where this replacement is best done, i.e. would it be better if done in the `frontend` or the `api`, rather than the `pg` container?

Reply:
I have no problem of keeping it here. It gives a strong connection between the replacement and where it is actually used

Reply:
I also think it's good to place the replacement here. I was a bit confused by the commit message at the beginning though. Sounds like the search term after replacement is used in sql querying but actually the modified strings are used for similarity calculation.